### DISCUSSION TITLE
(Feature request) be able to have back/forward above note text

### DISCUSSION THREAD

**Discussion by randomname42** [TriliumNext/Trilium#7337] [Q&A] [answered]
### Describe feature

~~I just started checking out Trilium and it looks like it could be pretty useful. But there's something that's really bothering me. I'm noticing right away that I keep wanting to be able to go back to the page I was previously on very often after clicking on something. The lack of back and forward buttons are making it very hard to get into using Triluim.~~

~~To be honest I don't think I could ever get used to Trilium without there being back/forward functionality.~~

### Additional Information

Edit: I found the option to add those, but I'm confused why they aren't shown by default. As a first time user my initial impression was that there was no back/forward functionality.

It was also an accident that I discovered this. I hadn't moused over the Triluim leaf logo in the upper right because it's just the program logo and it looked like decoration so much that my brain never even considered it might be anything more than decoration (hidden in plain sight, I guess).

I accidentally moved the mouse over the leaf and saw it get colors. Then I realized that's where the options and launchbar settings are found. From there it was easy to figure out how to add those buttons.

I still don't understand why those buttons are hidden by default, however. They are very useful. I've noticed there's some keyboard shortcuts too, but when I'm clicking on links I'm using the mouse already anyway so it's more convenient to click a button than to take my hand away from the mouse for a two-hand shortcut.

What I'd really like is to be able to **put the back/forward buttons on the same line as Basic Properties**, Owned Attributes, etc. That would be more conveniently positioned (above the note text) than all the way to the side of the program.

**Comment by rom1dep** [comment, marked as answer]
Click on the trilium icon and then "Configure Launchbar",  from there you can add those buttons by moving them to the visible section. I personally use keyboard shortcuts for that, which you can equally configure.

**Comment by zerebos** [comment]
If you want to put it above the note text area take a look at [Trilium Breadcrumbs](https://github.com/rauenzi/Trilium-Breadcrumbs)

**Comment by randomname42** [comment]
Thank you. That's a definite improvement. Navigating feels much nicer now. 

Because Trilium Breadcrumbs adds an extra row that takes away some of the vertical viewing space from notes, being able to have back/forward on the same row as Basic Properties and such (thereby avoiding taking away any vertical space from notes) would still be preferable however, if it could be done.

I did figure out how to minimize the vertical size of the breadcrumbs by reducing the padding. If there's a way to move it into the Basic Properties row, I don't know how. I'm guessing that row is not something addons can modify (because it's a built in part of the program)?

I think I understand how to just have only forward/back show up, so the buttons should fit on that row if it's possible to move them there.

**Comment by zerebos** [comment]
It is possible to do in theory with css, but I am away from home and unable to provide a personal solution at this time.
